c/tskit/tables.c

```diff
@@ -927,8 +930,8 @@ tsk_node_table_copy(tsk_node_table_t *self, tsk_node_table_t *dest, tsk_flags_t
 
 int TSK_WARN_UNUSED
 tsk_node_table_set_columns(tsk_node_table_t *self, tsk_size_t num_rows,
-    tsk_flags_t *flags, double *time, tsk_id_t *population, tsk_id_t *individual,
-    const char *metadata, tsk_size_t *metadata_offset)
+    tsk_flags_t *flags, const double *time, const tsk_id_t *population,
```

Comment:
flags should be const here I think

Reply:
Looks like there's a good few of these - I won't comment on any more of them

Reply:
Makes sense to me too to have these be const too. I suspect I skipped them since they were listed as "Undocumented methods" in the header file. But seeing now what all has been updated, it does seem like it will be less confusing to just have everything in the file be using const when appropriate rather.

I'll go through and update more.

Reply:
Good catches, does not make sense why I omitted to const those. Fixes in this commit now. I'll comment on another section which I've skipped. And inclined to skip for now.